sdc/tests/test_series.py

```diff
@@ -4737,12 +4738,12 @@ def test_impl(S, ind, val):
 
         with self.assertRaises(TypingError) as raises:
             hpat_func(S, ind1, value1)
-        msg = 'TypingError: Operator setitem(). The object value'
+        msg = 'TypingError: Operator setitem. The object value'
```

Comment:
I think `TypingError: ` is excess in the message. Moreover let's provide full message (what's given, what's expected).

Reply:
This problem, because message will be not in 1 string,  this message create error.